core/src/main/scala/cell/Cell.scala

```diff
@@ -37,7 +37,7 @@ trait Cell[K <: Key[V], V] {
    * }}}
    *
    * @param other  Cell that `this` Cell depends on.
-   * @param valueCallback  Callback that receives the final value of `other` and returns an `Outcome` for `this` cell.
+   * @param valueCallback  Callback that retrieves the final value of `other` and returns an Outcome for `this` cell.
```

Comment:
Not sure this is an improvement. ;-) When the callback is invoked, I would say that it "receives" the final value. "Retrieving" in my mind would require the callback in its body to call a "get" method or similar.

Reply:
Reverted this.